src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.